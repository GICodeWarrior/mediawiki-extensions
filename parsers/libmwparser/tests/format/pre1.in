test text ''
  indentet ''' format
    still indent

  new indent
* list

  new indent
{|
   | col1

    indent
|}


   indent