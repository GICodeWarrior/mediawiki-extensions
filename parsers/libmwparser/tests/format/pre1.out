BEGIN ARTICLE
    BEGIN PARAGRAPH
        WORD[test]
        SPACE[ ]
        WORD[text]
        SPACE[ ]
    END PARAGRAPH
    BEGIN PRE
        WORD[indentet]
        SPACE[ ]
        BEGIN BOLD
            SPACE[ ]
            WORD[format]
        END BOLD
        WORD[still]
        SPACE[ ]
        WORD[indent]
    END PRE
    BEGIN PRE
        WORD[new]
        SPACE[ ]
        WORD[indent]
    END PRE
    BEGIN BULLET LIST
        BEGIN BULLET LIST ITEM
            WORD[list]
        END BULLET LIST ITEM
    END BULLET LIST
    BEGIN PRE
        WORD[new]
        SPACE[ ]
        WORD[indent]
    END PRE
    BEGIN TABLE
        BEGIN TABLE BODY
            BEGIN ROW
                BEGIN CELL
                    SPACE[ ]
                    WORD[col]
                    WORD[1]
                    NEWLINE
                    NEWLINE
                    BEGIN PRE
                        WORD[indent]
                    END PRE
                END CELL
            END ROW
        END TABLE BODY
    END TABLE
    BEGIN PRE
        WORD[indent]
    END PRE
END ARTICLE
